meshtastic/mesh_interface_runtime/request_wait.py

```
@@ -0,0 +1,487 @@
+"""Request wait runtime for managing async response handling."""
+
+from __future__ import annotations
+
+import logging
+import threading
+import time
+from typing import Any, Callable, Optional
```

Ruff (0.15.7)

[warning] Import from `collections.abc` instead: `Callable`

Import from `collections.abc`

(UP035)